src/sys/windows/mod.rs

```diff
@@ -8,6 +8,8 @@ use winapi::shared::minwindef::{BOOL, HKEY};
 use winapi::shared::winerror;
 use winapi::um::{commapi, fileapi, handleapi, ioapiset, minwinbase, synchapi, winbase, winnt, winreg};
 
+const DEFAULT_TIMEOUT: u32 = 10;
```

Comment:
As a default timeout, I think 10ms is too short. Normally read/writes can block for much longer before timing out. Maybe even a few seconds would be good.

The value of 10 ms was the timeout between receiving character data, where it could be sensible.

Reply:
This time out, at least in POSIX land is how we wait for data to be available, or block on the serial port waiting to write some data. Usually its basically 0 for serial ports. Usually you try and read/write data immediately without  a timeout, and the system tells you how many bytes you actually read/wrote. 

I know windows has its own weird windows-isms, so I don't know.

Reply:
I think a lot of people doing serial on windows get confused because many other packages on windows ( pyserial, arduino ) basically configure the windows serial port handling to be more like POSIX.

Reply:
I'm also in favor of emulating POSIX behaviour, but I still think 10ms is very short. On Linux, the default timeout for a virtual terminal seems to be infinite (or at least longer than my patience).

Note that the timeout only triggers if *no data* is available. If data is available, it will still return without waiting for the timeout. That's why I think 10ms is too short as a default timeout.